librosa/core/intervals.py

```diff
@@ -2,21 +2,38 @@
 # -*- encoding: utf-8 -*-
 """Functions for interval construction"""
 
+from typing import Collection, Dict, List, Union, overload
+from typing_extensions import Literal
 import msgpack
 from pkg_resources import resource_filename
 import numpy as np
+from numpy.typing import ArrayLike
 from .._cache import cache
 
 
+__all__ = [
+    "INTERVALS",
+    "interval_frequencies",
+    "pythagorean_intervals",
+    "plimit_intervals",
+]
```

Comment:
There is no `intervals.pyi` so if someone does `from librosa.core.intervals import *` they would also import `fdesc`, which is untyped..... Pyright was complaining about an untyped public member, that's why I added the `__all__` here

Reply:
Hrm, it shouldn't need its own submodule .pyi if it's all specified in `__init__.pyi` right?

Reply:
Each module stands on its own (you can do `from librosa.core.intervals import *`).
This is a small and obscure detail though, if you prefer to remove the `__all__` for some reason, that's ok with me.

Reply:
> Each module stands on its own (you can do from librosa.core.intervals import *).

Right, I think this is more of a style question than anything else.  The API is designed so that all core functionality (regardless of subsubmodule) is exported through core (and all core functionality is exposed at the top level).  A user shouldn't need to explicitly import core.intervals.

Of course, adding `__all__` here will make pyright happier when working with individual submodules, but I'd generally prefer to not add all's back into submodules unless absolutely necessary.

> Pyright was complaining about an untyped public member (fdesc), that's why I added the __all__ here

fdesc really shouldn't have been interpreted as a public member; it's a one-time use variable needed for loading the interval mapping at import time.  If this was the only reason for adding all here, I think it would be better to rename fdesc (`__fdesc`) so that it's not interpreted as public.

Reply:
OK, then let's rename fdesc then

Reply:
Done